app/lib/features/home/pages/dashboard.dart

```diff
@@ -28,104 +26,196 @@ class Dashboard extends ConsumerWidget {
+        child: Row(
+          children: [
+            const Icon(Icons.search),
+            const SizedBox(width: 8),
+            Text(L10n.of(context).search),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Widget homeQuickActions(BuildContext context) {
+    return Column(
+      children: [
+        const SizedBox(height: 20),
+        Row(
+          children: [
+            homeQuickActionItem(
+              context: context,
+              title: L10n.of(context).pins,
+              iconData: Atlas.pin,
+              color: Colors.orangeAccent,
+              onTap: () => context.pushNamed(Routes.pins.name),
+            ),
+            const SizedBox(width: 20),
+            homeQuickActionItem(
+              context: context,
+              title: L10n.of(context).events,
+              iconData: Atlas.calendar_dots,
+              color: Theme.of(context).colorScheme.primary,
+              onTap: () => context.pushNamed(Routes.calendarEvents.name),
+            ),
+          ],
+        ),
+        const SizedBox(height: 20),
+        Row(
+          children: [
+            homeQuickActionItem(
+              context: context,
+              title: L10n.of(context).tasks,
+              iconData: Atlas.list,
+              color: Colors.green,
+              onTap: () => context.pushNamed(Routes.tasks.name),
+            ),
+            const SizedBox(width: 20),
+            homeQuickActionItem(
+              context: context,
+              title: L10n.of(context).updates,
+              iconData: Atlas.megaphone_thin,
+              color: Colors.blueGrey,
+              onTap: () => context.goNamed(Routes.updates.name),
+            ),
+          ],
+        ),
+      ],
+    );
+  }
+
+  Widget homeQuickActionItem({
+    required BuildContext context,
+    required String title,
+    required IconData iconData,
+    required Color color,
+    required Function()? onTap,
+  }) {
+    return Expanded(
+      child: InkWell(
+        onTap: onTap,
+        child: Container(
+          padding: const EdgeInsets.all(12),
+          decoration: BoxDecoration(
+            color: Theme.of(context).colorScheme.surface,
+            borderRadius: const BorderRadius.all(Radius.circular(16)),
+          ),
+          child: Row(
+            children: [
+              Container(
+                padding: const EdgeInsets.all(10),
+                decoration: BoxDecoration(
+                  color: color,
+                  borderRadius: const BorderRadius.all(Radius.circular(100)),
+                ),
+                child: Icon(iconData, size: 16),
               ),
-              ...children,
+              const SizedBox(width: 8),
+              Text(title),
             ],
           ),
         ),
       ),
     );
   }
+
+  Widget myTaskList(BuildContext context, WidgetRef ref) {
+    final provider = ref.watch(featuresProvider);
+    bool isActive(f) => provider.isActive(f);
+    if (isActive(LabsFeature.tasks)) {
```

Comment:
if it isn't reused, we can simplify this call.

```suggestion
    if (ref.watch(featuresProvider.select((provider) => provider.isActive(LabsFeature.tasks))) {
```